script.py

```diff
@@ -0,0 +1,60 @@
+from pdf_parser import *
+from png_layout import *
+from ocr_parser import *
+import argparse
+
+file_small = "CTX200903109.pdf"
+file_big = "CTX200502635.pdf"
+
+pages_output_folder = os.environ['FOLDER_PAGES']
+layout_output_folder = os.environ['FOLDER_LAYOUTS']
+ocr_output_folder = os.environ['FOLDER_TEXTS']
+ocr_output_folder_gcv = ocr_output_folder + "_gcv"
+
+folder_brno = "/lnet/work/people/lutsai/atrium/Brno-20240119T165503Z-001/Brno"
+folder_arup = "/lnet/work/people/lutsai/atrium/ATRIUM_ARUP_vzorek/CTX"
+
+if __name__ == "__main__":
```

Comment:
See IV.4 (create a `main` function and move most of the code there.)

Reply:
fixed